app/townsquare/templates/townsquare/index.html

```diff
@@ -178,6 +178,13 @@
         </div>
       </div>
     </div>
+    <div>
+      <div class="custom-control custom-switch">
```

Comment:
can we fix the indentation? not sure if we have one closing "div" left

Reply:
Yes, we had. Fixed